konfera/models/abstract.py

```diff
@@ -0,0 +1,20 @@
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+from django.core.exceptions import ValidationError
+
+
+class FromToModel(models.Model):
+
+    class Meta:
+        abstract = True
+
+    date_from = models.DateTimeField(verbose_name=_('Beginning'))
+    date_to = models.DateTimeField(verbose_name=_('End'))
+
+    def clean(self):
+        if self.date_from >= self.date_to:
```

Comment:
`TypeError at /admin/konfera/event/add/
unorderable types: NoneType() >= NoneType()` when a blank form is submitted